t04_refactored.py

```diff
@@ -120,138 +120,203 @@ def team_1_adv():
     pass
     # TODO Add your code here
 
+
 ###################################################################################
 
 def team_2_adv():
     pass
     # TODO Add your code here
 
+
 ###################################################################################
 
 def team_3_adv():
     pass
     # TODO Add your code here
 
+
 ###################################################################################
 
 def team_4_adv():
     pass
     # TODO Add your code here
 
+
 ###################################################################################
 
 def team_5_adv():
     pass
     # TODO Add your code here
 
+
 ###################################################################################
 
 def team_6_adv():
     pass
     # TODO Add your code here
 
+
 ###################################################################################
 
 def team_7_adv():
     pass
     # TODO Add your code here
 
+
 ###################################################################################
 
 def team_8_adv():
     pass
     # TODO Add your code here
 
+
 ###################################################################################
 
 def team_9_adv():
     pass
     # TODO Add your code here
 
+
 ###################################################################################
 
 def team_10_adv():
     pass
     # TODO Add your code here
 
+
 ###################################################################################
 
 def team_11_adv():
     pass
     # TODO Add your code here
 
+
 ###################################################################################
 
 def team_12_adv():
     pass
     # TODO Add your code here
 
+
 ###################################################################################
 
 def team_13_adv():
-    pass
-    # TODO Add your code here
+    """
+    https://docs.google.com/document/d/1Dl_uUgAPFY98r5OVFB7_ctGqmgeOoFKrI5tu8xMKJ2w/edit?usp=sharing
+    Michael Vargas
+    Zach Anderson
+    :return: none
+    """
+
+    from time import sleep
```

Comment:
Line 22 has this import already, so you don't need it here. PyCharm is probably hinting at that.